src/test/java/apicurito/tests/steps/verification/CommonVerifications.java

```diff
@@ -0,0 +1,104 @@
+package apicurito.tests.steps.verification;
+
+import apicurito.tests.utils.slenide.CommonUtils;
+import apicurito.tests.utils.slenide.OperationUtils;
+import apicurito.tests.utils.slenide.PathUtils;
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+import cucumber.api.java.en.Then;
+import io.cucumber.datatable.DataTable;
+import org.openqa.selenium.By;
+
+import java.util.List;
+
+import static com.codeborne.selenide.Condition.text;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+public class CommonVerifications {
+
+    private static class Elements {
+        private static By QUERY_PARAM_SECTION = By.cssSelector("query-params-section");
+        private static By HEADER_PARAM_SECTION = By.cssSelector("header-params-section");
+        private static By REQUEST_BODY_SECTION = By.cssSelector("requestbody-section");
+        private static By DESCRIPTION = By.className("description");
+        private static By PARAMETERS_TYPE = By.className("param-type");
+    }
+
+    /*
+        param could be header|query|RFD
+        page could be path|operations
+     */
+    @Then("check that exist {string} parameters or RDF on {string} page")
+    public void checkThatExistParametersOnPage(String param, String page, DataTable table) {
+        SelenideElement pageElement = page.equals("operations") ? OperationUtils.getOperationRoot() : PathUtils.getPathPageRoot();
+        String rowType = null;
+        By section = null;
+
+        switch (param) {
```

Comment:
Also this switch was already once specified above - could it be created in a more reusable way?

Reply:
It will be part of the next PR as we discussed.